In [7]:
txt = """
Для нахождения процента израсходованного материала, используем формулу: $$ \\text{Процент} = \\left( \\frac{\\text{израсходовано}}{\\text{общая длина}} \\right) \\cdot 100 \\% $$ 1. Подставим известные значения в формулу:  Израсходовано = 210 метров, Общая длина = 450 метров.  $$ \\text{Процент} = \\left( \\frac{210}{450} \\right) \\cdot 100 \\% $$ 2. Вычислим дробь: $$ \\frac{210}{450} = 0.4667 $$ 3. Умножим на 100, чтобы получить процент: $$ 0.4667 \\cdot 100 = 46.67 \\% $$ 4. Округлим до двух знаков после запятой: 46.67%. Таким образом, на изготовление курток израсходовали 46.67% материала.
"""

In [ ]:
print(txt)


Для нахождения процента израсходованного материала, используем формулу: $$ \text{Процент} = \left( \frac{\text{израсходовано}}{\text{общая длина}} \right) \cdot 100 \% $$ 1. Подставим известные значения в формулу:  Израсходовано = 210 метров, Общая длина = 450 метров.  $$ \text{Процент} = \left( \frac{210}{450} \right) \cdot 100 \% $$ 2. Вычислим дробь: $$ \frac{210}{450} = 0.4667 $$ 3. Умножим на 100, чтобы получить процент: $$ 0.4667 \cdot 100 = 46.67 \% $$ 4. Округлим до двух знаков после запятой: 46.67%. Таким образом, на изготовление курток израсходовали 46.67% материала.



: 


Для нахождения процента израсходованного материала, используем формулу: $$ \text{Процент} = \left( \frac{\text{израсходовано}}{\text{общая длина}} \right) \cdot 100 \% $$ 1. Подставим известные значения в формулу:  Израсходовано = 210 метров, Общая длина = 450 метров.  $$ \text{Процент} = \left( \frac{210}{450} \right) \cdot 100 \% $$ 2. Вычислим дробь: $$ \frac{210}{450} = 0.4667 $$ 3. Умножим на 100, чтобы получить процент: $$ 0.4667 \cdot 100 = 46.67 \% $$ 4. Округлим до двух знаков после запятой: 46.67%. Таким образом, на изготовление курток израсходовали 46.67% материала.


 $138: \bigcirc=\square \%-138$ метров составляют $\square \%$ от 230 метров

В $216$ метрах содержится $\bigcirc$ метров, что составляет $\square \%$ от $480$ метров. Найдите значение $\square$.


Для решения задачи нужно сначала найти, что такое $\bigcirc$. Мы знаем, что $\bigcirc = 216$ метров. Теперь мы можем выразить процентное соотношение. Сначала зададим уравнение: \[216 = \frac{\square}{100} \cdot 480.\]\
Чтобы найти $\square$, умножим обе стороны уравнения на $100$: \[21600 = \square \cdot 480.\]\
Теперь разделим обе стороны на $480$: \[\square = \frac{21600}{480}.\]\
Выполним деление: \[\square = 45.\] Таким образом, $\square$ составляет $45\%$.

230: 100=\bigcirc(м)-1 \\% от 230 метров.

# Реализуем функций валидирующие наши parsed_tasks и generated_new_tasks

In [23]:
import json
import re

from IPython.core.display import Latex
from fontTools.afmLib import readlines

In [24]:
tasks_path = "parsed_task_dir/algebra_6_parsed_task_from_md.jsonl"
new_tasks_path = "generated_tasks_dir/algebra_6_1_gen_parsed_tasks.jsonl"

In [25]:

def collapse_backslashes(text: str) -> str:
    """
    Заменяет \\  \\\\  \\\\\\ … на одинарный \.
    Работает безопасно для всего текста: формул, JSON-фрагментов и т.д.
    """
    return re.sub(r'\\{2,}', r'\\', text)

In [35]:
collapse_backslashes("\\frac")

'\\frac'

In [26]:
tasks = []

with open(tasks_path, "r", encoding="utf-8") as f:
    for i, line in enumerate(f, 1):
        task = json.loads(line)
        task = json.loads(task) # превращаем строку в dict
        print(task)
        tasks.append(task)
        print(50*"+")

{'id': '1', 'original': 'Заменив звездочку соответствующей цифрой, заполните таблицу.\n\n|  | Числа, <br> делящиеся <br> на 2 | Числа, <br> делящиеся <br> на 5 | Числа, <br> делящиеся <br> на 10 | Числа, <br> делящиеся <br> на 3 | Числа, <br> делящиеся <br> на 9 |\n| :---: | :---: | :---: | :---: | :---: | :---: |\n| $49^{*}$ |  |  |  |  |  |\n| $83^{*}$ |  |  |  |  |  |', 'solution': None, 'answer': None, 'topic': 'Делимость натуральных чисел', 'difficulty': 'A', 'tags': ['делимость', 'натуральные числа', 'признаки делимости', 'таблица', 'заполнение', 'цифры', 'числа с пропусками'], 'type': 'Практическая', 'valid': True, 'remark': None}
++++++++++++++++++++++++++++++++++++++++++++++++++
{'id': 'II. Задача 2', 'original': 'Из чисел $1,2,6,7,13,15,20,41,49$ выпишите числа, имеющие:\n\n1) только один делитель;\n2) только два делителя;\n3) более двух делителей.', 'solution': None, 'answer': None, 'topic': 'Делимость натуральных чисел', 'difficulty': 'B', 'tags': ['делители', 'натуральные 

In [38]:
new_tasks = []

with open(new_tasks_path, "r", encoding="utf-8") as f:
    for i, line in enumerate(f, 1):
        task = json.loads(line)
        task["question"] = collapse_backslashes(task["question"].strip())
        task["solution"] = collapse_backslashes(task["solution"].strip())
        task["answer"] = collapse_backslashes(task["answer"].strip())
        print(task)
        new_tasks.append(task)
        print(50*"+")

{'question': 'Найдите наименьшее натуральное число, которое делится на 3, 4 и 5 одновременно.', 'solution': 'Чтобы найти наименьшее натуральное число, которое делится на 3, 4 и 5 одновременно, нужно найти наименьшее общее кратное (НОК) этих чисел. Разложим числа на простые множители:\\n\\n- 3 = 3\\n- 4 = 2^2\\n- 5 = 5\\n\\nНОК будет содержать каждый простой множитель в максимальной степени:\\n\\n$\\text{НОК} = 2^2 \\cdot 3 \\cdot 5 = 4 \\cdot 3 \\cdot 5 = 60$.\\n\\nНаименьшее число, которое делится на 3, 4 и 5 одновременно, равно 60.', 'topic': 'Делимость натуральных чисел', 'answer': '60', 'type': 'Практическая', 'tags': ['делимость', 'натуральные числа', 'НОК', 'простые множители'], 'difficulty': 'A'}
++++++++++++++++++++++++++++++++++++++++++++++++++
{'question': 'Найдите наибольший общий делитель (НОД) чисел 84 и 126.', 'solution': 'Мы начнем с разложения чисел на простые множители. \\n Для 84: \\n $84 = 2^2 \\cdot 3 \\cdot 7$.\\n Для 126: \\n $126 = 2 \\cdot 3^2 \\cdot 7$. \\n Зат

In [39]:
# %% 🛠️ Быстрая проверка Markdown + LaTeX
import markdown, re
from pylatexenc.latexwalker import LatexWalker, LatexWalkerParseError

def quick_md_check(text: str) -> list[str]:
    """Возвращает список найденных проблем."""
    probs = []
    txt = collapse_backslashes(text)

    # непарные ```  или $
    if txt.count("```") % 2: probs.append("непарное число ```")
    if txt.count("$")   % 2: probs.append("непарное число $")

    # заголовки без пробела
    if re.search(r"^(#{1,6})(\S)", txt, flags=re.MULTILINE):
        probs.append("нет пробела после # в заголовке")

    # валидируем LaTeX-фрагменты
    for formula in re.findall(r"\$(.*?)\$", txt):
        try:
            LatexWalker(formula).get_latex_nodes()
        except LatexWalkerParseError as e:
            probs.append(f"LaTeX: {e}")

    # markdown → html (ловим SyntaxError от парсера)
    try:
        markdown.markdown(txt, extensions=["fenced_code", "tables"])
    except Exception as e:
        probs.append(f"Markdown-parser: {e}")
    return probs


In [41]:
# %% 🏃‍♂️ Главная функция для списка задач
from termcolor import colored
from tqdm import tqdm

def validate_tasks(tasks, quest_key="question"):
    bad = 0
    for i, t in enumerate(tqdm(tasks, desc="Проверка")):
        md = t[quest_key]
        errs = quick_md_check(md)
        if errs:
            bad += 1
            print(colored(f"[{i}] ❌ {errs}", "red"))
        else:
            print(colored(f"[{i}] ✅ OK", "green"))
    print(colored(f"\nИТОГО: ошибок {bad} из {len(tasks)}", "cyan"))

In [42]:
validate_tasks(new_tasks)

Проверка: 100%|██████████| 33/33 [00:00<00:00, 500.69it/s]

[0] ✅ OK
[1] ✅ OK
[2] ✅ OK
[3] ✅ OK
[4] ✅ OK
[5] ✅ OK
[6] ✅ OK
[7] ✅ OK
[8] ✅ OK
[9] ✅ OK
[10] ✅ OK
[11] ✅ OK
[12] ✅ OK
[13] ✅ OK
[14] ✅ OK
[15] ✅ OK
[16] ✅ OK
[17] ✅ OK
[18] ✅ OK
[19] ✅ OK
[20] ✅ OK
[21] ✅ OK
[22] ✅ OK
[23] ✅ OK
[24] ✅ OK
[25] ✅ OK
[26] ✅ OK
[27] ✅ OK
[28] ✅ OK
[29] ✅ OK
[30] ✅ OK
[31] ✅ OK
[32] ✅ OK

ИТОГО: ошибок 0 из 33


In [43]:
validate_tasks(new_tasks)

Проверка: 100%|██████████| 33/33 [00:00<00:00, 2366.26it/s]

[0] ✅ OK
[1] ✅ OK
[2] ✅ OK
[3] ✅ OK
[4] ✅ OK
[5] ✅ OK
[6] ✅ OK
[7] ✅ OK
[8] ✅ OK
[9] ✅ OK
[10] ✅ OK
[11] ✅ OK
[12] ✅ OK
[13] ✅ OK
[14] ✅ OK
[15] ✅ OK
[16] ✅ OK
[17] ✅ OK
[18] ✅ OK
[19] ✅ OK
[20] ✅ OK
[21] ✅ OK
[22] ✅ OK
[23] ✅ OK
[24] ✅ OK
[25] ✅ OK
[26] ✅ OK
[27] ✅ OK
[28] ✅ OK
[29] ✅ OK
[30] ✅ OK
[31] ✅ OK
[32] ✅ OK

ИТОГО: ошибок 0 из 33


In [46]:
from IPython.core.display import Markdown

for task in new_tasks:
    question = collapse_backslashes(task["question"])
    solution = collapse_backslashes(task["solution"])

    res = question + "\n\n" + solution + "\n" + 100*"~" + "\n\n\n"
    display(Markdown(res))

Найдите наименьшее натуральное число, которое делится на 3, 4 и 5 одновременно.

Чтобы найти наименьшее натуральное число, которое делится на 3, 4 и 5 одновременно, нужно найти наименьшее общее кратное (НОК) этих чисел. Разложим числа на простые множители:\n\n- 3 = 3\n- 4 = 2^2\n- 5 = 5\n\nНОК будет содержать каждый простой множитель в максимальной степени:\n\n$\text{НОК} = 2^2 \cdot 3 \cdot 5 = 4 \cdot 3 \cdot 5 = 60$.\n\nНаименьшее число, которое делится на 3, 4 и 5 одновременно, равно 60.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Найдите наибольший общий делитель (НОД) чисел 84 и 126.

Мы начнем с разложения чисел на простые множители. \n Для 84: \n $84 = 2^2 \cdot 3 \cdot 7$.\n Для 126: \n $126 = 2 \cdot 3^2 \cdot 7$. \n Затем мы находим общие множители и перемножаем их с наименьшими показателями: \n Общий множитель для $2$: $2^1$, \n Общий множитель для $3$: $3^1$, \n Общий множитель для $7$: $7^1$. \n \n Перемножим общие множители: \n $2^1 \cdot 3^1 \cdot 7^1 = 42$. \n Таким образом, наибольший общий делитель (НОД) чисел 84 и 126 равен 42.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Определите, делится ли число 47325 на 3 и 9 без остатка.

Для проверки делимости числа на 3 необходимо найти сумму его цифр и проверить, делится ли она на 3. \[ 4 + 7 + 3 + 2 + 5 = 21 \] Поскольку 21 делится на 3, то число 47325 делится на 3.\n\nДля проверки делимости числа на 9 необходимо также найти сумму его цифр и проверить, делится ли она на 9. \[ 21 \] Поскольку 21 не делится на 9, число 47325 не делится на 9.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Докажите, что если натуральное число $a$ делится на 3, то квадрат этого числа $a^2$ также делится на 3.

Пусть $a$ - натуральное число, такое что $a \equiv 0 \pmod{3}$. Это означает, что существует некоторое целое число $k$, для которого $a = 3k$. Тогда $a^2 = (3k)^2 = 9k^2 = 3(3k^2)$. Поскольку $3k^2$ является целым числом, можно сделать вывод, что $a^2$ делится на 3.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Докажите, что для любого простого числа $p > 2$, число $p^2 - 1$ делится на 24.

Рассмотрим простое число $p$ больше 2. Тогда оно должно быть нечётным. Следовательно, $p$ можно представить в виде $p = 2k + 1$, где $k$ — целое число.\n\nТеперь найдём $p^2 - 1$:\n\n$$p^2 - 1 = (2k + 1)^2 - 1 = 4k^2 + 4k + 1 - 1 = 4k(k + 1).$$\n\nЗаметим, что произведение $k(k + 1)$ всегда чётно (одно из чисел обязательно чётное). Следовательно, $4k(k + 1)$ делится на 8.\n\nКроме того, среди трех последовательных чисел $p - 1$, $p$, $p + 1$ одно число делится на 3 (из-за делимости по модулю 3). Но числа $p - 1$ и $p + 1$ оба чётные и делятся на 2. Таким образом, хотя бы одно из произведений $(p - 1)/2$ и $(p + 1)/2$ делится на 3. Следовательно,  $p^2 - 1 = (p - 1)(p + 1)$ делится на 3.\n\nПоскольку $p^2 - 1$ делится на 8 и на 3, оно делится на их произведение, то есть на 24. \n\nТаким образом, доказано, что $p^2 - 1$ делится на 24.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Найдите наименьшее общее кратное чисел 24 и 36.

Чтобы найти наименьшее общее кратное (НОК) чисел 24 и 36, сначала разложим каждое из чисел на простые множители: \n\n $24 = 2^3 \cdot 3^1$ \n $36 = 2^2 \cdot 3^2$ \n\n Затем НОК будет равен произведению всех простых множителей, взятых с наибольшими показателями: \n\n $НОК = 2^3 \cdot 3^2 = 72$
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Найдите целое число \(x\), которое при делении на 5 дает остаток 2 и при делении на 7 дает остаток 1.

Пусть \(x = 5k + 2\) и \(x = 7m + 1\), где \(k\) и \(m\) — целые числа.\n\nПриравняем выражения для \(x\):\n\n\(5k + 2 = 7m + 1\)\n\nУпростим уравнение:\n\n\(5k - 7m = -1\)\n\nНайдем частное решение уравнения чисел путем подбора. \n\nБерем \(k = 3\), тогда:\n\n\(5 \cdot 3 - 7 \cdot 2 = 15 - 14 = 1\)\n\nКорректируем на нужный знак:\n\n\(5 \cdot 1 - 7 \cdot 1 = -1\) — корректное решение.\n\nТеперь общее решение:\n\n\(5k - 7m = -1\) — частное решение \(k = 1, m = 1\).\n\nОбщее решение:\n\n\(k = 1 + 7t\), \(m = 1 + 5t\), где \(t\) — любое целое число.\n\nПодставим в выражение для \(x\):\n\n\(x = 5k + 2 = 5(1 + 7t) + 2 = 5 + 35t + 2 = 7 + 35t\).\n\nПервое положительное число получится при \(t = 0\):\n\n\(x = 7\).
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Лодка движется по реке. Скорость течения реки составляет $2$ км/ч. Лодка проплывает $16$ км по течению и $9$ км против течения, затратив на это $5$ часов. Найдите скорость лодки в неподвижной воде.

Пусть $v$ км/ч – скорость лодки в неподвижной воде. Тогда скорость лодки по течению равна $v + 2$ км/ч, а против течения – $v - 2$ км/ч. \n\nДля движения по течению:\n\[ \frac{16}{v + 2} \] – время, затраченное на 16 км.\n\nДля движения против течения:\n\[ \frac{9}{v - 2} \] – время, затраченное на 9 км.\n\nОбщее время: \n\[ \frac{16}{v + 2} + \frac{9}{v - 2} = 5 \]\n\nУмножим на $(v + 2)(v - 2)$ и получим уравнение:\n\[ 16(v - 2) + 9(v + 2) = 5(v^2 - 4) \]\n\nРаскрываем скобки и приводим подобные:\n\[ 16v - 32 + 9v + 18 = 5v^2 - 20 \]\n\[ 25v - 14 = 5v^2 - 20 \]\n\[ 0 = 5v^2 - 25v - 6 \]\n\nРешим квадратное уравнение: \n\[ v^2 - 5v - 1.2 = 0 \]\n\nИспользуем формулу корней квадратного уравнения:\n\[ v = \frac{-b \pm \sqrt{b^2 - 4ac}}{2a} \]\nгде $a = 1$, $b = -5$, $c = -1.2$\n\n\[ v = \frac{5 \pm \sqrt{25 + 4.8}}{2} \]\n\[ v = \frac{5 \pm \sqrt{29.8}}{2} \]\n\n\[ v = \frac{5 \pm 5.46}{2} \]\nВыбираем положительный корень:\n\[ v = \frac{10.46}{2} = 5.23 \]\n\nСкорость лодки в неподвижной воде составляет $5.23$ км/ч.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Две шестерни сцеплены. У большой шестерни 64 зубьев, а у малой — 48. Найдите количество оборотов, которое должна сделать малая шестерня, чтобы оба зубья снова соприкоснулись в том же положении.

Пусть количество оборотов малой шестерни будет $y$, а большое шестерни — $x$. Поскольку зубья сцеплены, то: \n\n\[ 64x = 48y \]\n\nНайдем наименьшее общее кратное (НОК) чисел 64 и 48, чтобы определить минимальное число зубьев, после которого они совпадут.\n\nРазложим на простые множители:\n\n\[ 64 = 2^6, \quad 48 = 2^4 \cdot 3 \]\n\nНОК:\n\n\[ 2^6 \cdot 3 = 192 \]\n\nТогда:\n\n\[ 48y = 192 \Rightarrow y = 4 \]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Найдите наименьшее натуральное число, которое при делении на 4 даёт остаток 3 и при делении на 5 даёт остаток 2.

Обозначим искомое число через $n$. Тогда \( n \equiv 3 \pmod{4} \) и \( n \equiv 2 \pmod{5} \). Для решения этой системы уравнений применим метод подбора:\n\nПервое уравнение можно представить как $n = 4k + 3$, где $k$ — некоторое целое число. Подставим это в второе уравнение: \n\n$4k + 3 \equiv 2 \pmod{5}$.\n\nУпростим уравнение: $4k \equiv -1 \equiv 4 \pmod{5}$,\nчто эквивалентно $4k \equiv 4 \pmod{5}$.\n\nСледовательно, $k \equiv 1 \pmod{5}$. Разрешая это уравнение, получаем $k = 5m + 1$, где $m$ — целое число.\n\nПодставляем $k$ обратно в выражение для $n$:\n\n$n = 4(5m + 1) + 3 = 20m + 4 + 3 = 20m + 7$.\n\nМинимальное значение $m=0$ даёт нам $n = 7$. Проверка: $7 \div 4 = 1$ с остатком $3$ и $7\div 5 = 1$ с остатком $2$. Условия выполняются.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




На чертеже изображен квадрат со стороной $12 \, \text{см}$. Этот квадрат увеличили в 3 раза по каждому измерению. Найдите площадь увеличенного квадрата.

Площадь первоначального квадрата равна $12 \cdot 12 = 144 \, \text{см}^2$. При увеличении в 3 раза, сторона нового квадрата станет $3 \cdot 12 = 36 \, \text{см}$. Площадь нового квадрата равна $36 \cdot 36 = 1296 \, \text{см}^2$.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




В магазине сахар продается в двух упаковках. Маленькая упаковка содержит 800 г сахара и стоит 56 рублей. Большая упаковка весит 1.5 кг. Используя пропорцию, найдите стоимость большой упаковки, если цена за 1 г сахара одинаковая.

Стоимость 1 г сахара в маленькой упаковке: $\frac{56}{800} = 0.07$ руб/г. Вес большой упаковки в граммах: $1.5 \cdot 1000 = 1500$ г. Стоимость большой упаковки: $1500 \cdot 0.07 = 105$ рублей.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Упростите выражение: $\frac{x^2 - 4}{x^2 - 5x + 6} \cdot \frac{x - 3}{x + 2}$

Сначала разложим числитель и знаменатель первой дроби на множители.\n\nЧислитель: $x^2 - 4 = (x - 2)(x + 2)$.\n\nЗнаменатель: $x^2 - 5x + 6 = (x - 2)(x - 3)$.\n\nТеперь выразим всё выражение в виде произведения:\n\n$\frac{(x - 2)(x + 2)}{(x - 2)(x - 3)} \cdot \frac{x - 3}{x + 2}$.\n\nСократим выражение:\n\n1. $(x - 2)$ в числителе и знаменателе первой дроби.\n2. $(x + 2)$ в числителе первой дроби и знаменателе второй дроби.\n3. $(x - 3)$ в числителе второй дроби и знаменателе первой дроби.\n\nОстается 1 в числителе и знаменателе после сокращения всех возможных выражений.\n\nТаким образом, упрощенное выражение равно 1.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Смешайте однотонную краску и цветной пигмент, чтобы получить новую краску. Для создания определенного цвета требуется 2.5 литра однотонной краски и \( \frac{3}{4} \) литра цветного пигмента. Сколько литров будет в итоге получившейся краски?

Чтобы найти общее количество краски, нужно сложить однотонную краску и цветной пигмент. \n\nПереведем \( \frac{3}{4} \) в десятичную дробь: \n\[ \frac{3}{4} = 0.75 \]\n\nТеперь сложим: \n\[ 2.5 + 0.75 = 3.25 \]\n\nТаким образом, общее количество краски равно 3.25 литра.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Упростите выражение, содержащее как обыкновенные, так и десятичные дроби: $\frac{3}{5} + 0.8 - \frac{1}{4}$. Укажите конечный ответ в виде десятичной дроби.

Сначала преобразуйте обыкновенные дроби в десятичные дроби: $\frac{3}{5} = 0.6$ и $\frac{1}{4} = 0.25$. Далее выполните сложение и вычитание: $0.6 + 0.8 = 1.4$. Затем вычтите $0.25$ из $1.4$: $1.4 - 0.25 = 1.15$.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Переведите смешанное число $3 \frac{1}{4}$ в десятичную дробь и найдите сумму этого числа с числом $2.75$. Также укажите результат в виде несократимой обыкновенной дроби.

Сначала переводим $3 \frac{1}{4}$ в десятичную дробь. Для этого дробь $\frac{1}{4}$ нужно перевести в десятичную: $\frac{1}{4} = 0.25$. Теперь смешанное число $3 \frac{1}{4}$ становится $3.25$.\nТеперь находим сумму $3.25 + 2.75 = 6.00$. Сумма равна 6 в десятичной форме.\nТеперь представим это число в виде несократимой обыкновенной дроби: $6 = \frac{6}{1}$. Так как числитель и знаменатель не имеют общих простых множителей, кроме 1, то дробь и так несократимая.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Два человека работают вместе над проектом. Первый человек может завершить проект за $8$ дней, а второй за $6$ дней. Если они работают вместе и каждый день тратят одинаковое количество времени на работу, сколько дней у них займет завершение проекта?

Для начала найдем, какую часть проекта может сделать каждый человек за 1 день. Первый человек делает за 1 день $\frac{1}{8}$ проекта, второй делает за $1$ день $\frac{1}{6}$ проекта. Работая вместе, они делают за $1$ день $\frac{1}{8} + \frac{1}{6}$ проекта. Приведем дроби к общему знаменателю: \n\n$$\frac{1}{8} + \frac{1}{6} = \frac{3}{24} + \frac{4}{24} = \frac{7}{24}.$$\n\nТеперь найдем, за сколько дней они вместе выполнят весь проект, то есть $1$ проект. Это будет обратная величина суммы их работы за один день:\n\n$$\frac{1}{\frac{7}{24}} = \frac{24}{7}.$$ Это соответствует приблизительно $3.43$ дням.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Выполните следующие действия: прибавьте к числу $3.5$ дробь $\frac{1}{4}$, затем умножьте результат на $1.2$, и отнимите $\frac{2}{5}$. Какое число получилось в итоге?

1. Преобразуем дробь $\frac{1}{4}$ в десятичную: $0.25$. \n2. Прибавим $3.5 + 0.25 = 3.75$. \n3. Умножим результат на $1.2$: $3.75 \cdot 1.2 = 4.5$. \n4. Преобразуем $\frac{2}{5}$ в десятичную: $0.4$. \n5. Отнимем $4.5 - 0.4 = 4.1$.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Вычислите: $2 \frac{1}{2} + 3.75 - \frac{5}{8} \cdot 1.2 + \frac{4}{5} : 0.25$

Сначала преобразуем смешанные и обыкновенные дроби в десятичные: $2 \frac{1}{2} = 2.5$, $\frac{5}{8} = 0.625$, $\frac{4}{5} = 0.8$. Получаем числовое выражение: $2.5 + 3.75 - 0.625 \cdot 1.2 + 0.8 : 0.25$.\n\nВыполняем умножение: $0.625 \cdot 1.2 = 0.75$.\nВыполняем деление: $0.8 : 0.25 = 3.2$.\nВычисляем сумму и разности: $2.5 + 3.75 - 0.75 + 3.2 = 8.7$.\nИтак, ответ: $8.7$.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




В классе 60 учеников, из которых 30% играют в шахматы. Сколько учеников играют в шахматы?

Определим количество учеников, играющих в шахматы. Используем формулу для нахождения процента от числа: $ \text{Количество} = \frac{30}{100} \cdot 60 $. Получаем $ 18 $.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




У Кати и Маши в копилках находится определенное количество монет. Количество монет у Кати к количеству монет у Маши относится как 5 к 3. Если Кати дать ещё 12 монет, то у неё будет в 2 раза больше монет, чем у Маши. Сколько монет у каждой из них изначально?

Обозначим количество монет у Кати как $x$, а количество монет у Маши как $y$. Согласно условию задачи, мы имеем два уравнения:\n\n1. $\frac{x}{y} = \frac{5}{3}$ \text{ (отношение монет)}.\n\n2. $x + 12 = 2y$ \text{ (после добавления 12 монет, у Кати будет в два раза больше)}.\n\nРешим систему уравнений:\n\nС первого уравнения: $x = \frac{5}{3}y$.\n\nПодставим во второе уравнение:\n\n\[ \frac{5}{3}y + 12 = 2y \]\n\nУмножим все уравнение на 3, чтобы избавиться от дроби:\n\n\[ 5y + 36 = 6y \]\n\nРешая уравнение, получаем:\n\n\[ 36 = y \]\n\nТеперь найдем $x$:\n\n\[ x = \frac{5}{3} \cdot 36 = 60 \]\n\nТаким образом, у Кати изначально было 60 монет, а у Маши - 36 монет.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Два человека красят забор. Первый может покрасить весь забор за 6 часов, а второй за 8 часов. Сколько времени потребуется, чтобы вместе покрасить забор?

Первый человек за час красит $\frac{1}{6}$ части забора, а второй $\frac{1}{8}$. Вместе они красят $\frac{1}{6} + \frac{1}{8}$ части забора за час. Для нахождения общей части приведем дроби к общему знаменателю: $\frac{4}{24} + \frac{3}{24} = \frac{7}{24}$. \n\nТеперь, чтобы покрасить весь забор, понадобится $\frac{1}{\frac{7}{24}} = \frac{24}{7}$ часа. Это примерно 3 часа и 25 минут.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Сложите обыкновенную дробь $\frac{3}{4}$ и десятичную дробь $0.6$. Преобразуйте десятичную дробь в обыкновенную, выполните операцию и представьте ответ в виде десятичной дроби.

Сначала преобразуем десятичную дробь $0.6$ в обыкновенную дробь: $0.6 = \frac{6}{10}$. Упрощаем дробь: $\frac{6}{10} = \frac{3}{5}$. Теперь сложим дроби $\frac{3}{4}$ и $\frac{3}{5}$. Чтобы сложить дроби, приведем их к общему знаменателю. Наименьший общий знаменатель для 4 и 5 — это 20. Преобразуем дроби: \(\frac{3}{4} = \frac{3 \cdot 5}{4 \cdot 5} = \frac{15}{20}\) и \(\frac{3}{5} = \frac{3 \cdot 4}{5 \cdot 4} = \frac{12}{20}\). Теперь можем сложить: \(\frac{15}{20} + \frac{12}{20} = \frac{27}{20}\). Преобразуйте \(\frac{27}{20}\) в десятичную дробь. Делим 27 на 20: $27 \div 20 = 1.35$.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Сколько времени потребуется двум рабочим, чтобы выполнить заказ, если первый рабочий может выполнить работу за 5.5 часов, а второй - за \( \frac{12}{7} \) часов?

Пусть \( x \) - время, за которое оба рабочих выполнят заказ вместе. Тогда первый рабочий выполняет \( \frac{1}{5.5} \) части работы в час, а второй - \( \frac{7}{12} \) части работы в час. Составляем уравнение: \n\n$$ \frac{1}{5.5} + \frac{7}{12} = \frac{1}{x} $$\n\nПриводим дроби к общему знаменателю (66):\n\n$$ \frac{12}{66} + \frac{38.5}{66} = \frac{1}{x} $$\n\nСкладываем дроби:\n\n$$ \frac{50.5}{66} = \frac{1}{x} $$\n\nНаходим \( x \):\n\n$$ x = \frac{66}{50.5} \approx 1.307 $$\n\nОба рабочих вместе выполнят заказ примерно за 1.31 часа.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Лодка плывёт по реке с постоянной скоростью относительно воды. Скорость течения реки составляет 4 км/ч. Лодка проплыла 12 км по течению и 12 км против течения за 4 часа. Найдите скорость лодки относительно воды.

Пусть $v$ км/ч – скорость лодки относительно воды. Тогда скорость лодки по течению равна $(v + 4)$ км/ч, а против течения $(v - 4)$ км/ч. Используя формулу времени $t = \frac{s}{v}$, получаем $t_1 = \frac{12}{v + 4}$ и $t_2 = \frac{12}{v - 4}$. Сумма времени на пути составила 4 часа: $\frac{12}{v+4} + \frac{12}{v-4} = 4$. Объединив это уравнение и решив его, получим: $\frac{12}{v+4} + \frac{12}{v-4} = 4 \Rightarrow 12(v-4) + 12(v+4) = 4(v^2-16) \Rightarrow 24v = 4(v^2-16) \Rightarrow v^2 - 6v - 16 = 0$. Решая квадратное уравнение, находим $v = 8$ км/ч.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Теплоход отправился в поездку против течения реки из пункта A в пункт B и обратно. Скорость течения реки составляет 3 км/ч, а собственная скорость теплохода - 15 км/ч. Расстояние между пунктами A и B равно 36 км. Найдите общее время, затраченное на весь путь.

Обозначим скорость теплохода в стоячей воде за $v = 15$ км/ч и скорость течения реки за $u = 3$ км/ч. Тогда скорость теплохода против течения равна $v - u = 15 - 3 = 12$ км/ч, а по течению - $v + u = 15 + 3 = 18$ км/ч. Расстояние между пунктами $A$ и $B$ равно 36 км. \n\nВремя пути против течения: \n$$t_{\text{против}} = \frac{36}{12} = 3 \text{ часа}$$\nВремя пути по течению: \n$$t_{\text{по}} = \frac{36}{18} = 2 \text{ часа}$$\n\nОбщее время на весь путь: \n$$t_{\text{общий}} = 3 + 2 = 5 \text{ часов}$$
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Докажите, что сумма трех последовательных натуральных чисел всегда делится на 3.

Обозначим три последовательных натуральных числа как $n$, $n+1$ и $n+2$. Тогда их сумма равна $n + (n+1) + (n+2) = 3n + 3$. Это выражение можно записать как $3(n+1)$. Очевидно, что $3(n+1)$ делится на 3, так как это произведение 3 и целого числа $(n+1)$. Следовательно, сумма трех последовательных натуральных чисел всегда делится на 3.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Сложите обыкновенную дробь $\frac{3}{5}$ и десятичную дробь $0.8$, а затем результат разделите на $\frac{1}{2}$.

1. Приведем $0.8$ к обыкновенной дроби: $0.8 = \frac{8}{10} = \frac{4}{5}$. \n2. Сложим дроби $\frac{3}{5}$ и $\frac{4}{5}$: \n$\frac{3}{5} + \frac{4}{5} = \frac{3+4}{5} = \frac{7}{5}$. \n3. Разделим $\frac{7}{5}$ на $\frac{1}{2}$: \n$\frac{7}{5} \div \frac{1}{2} = \frac{7}{5} \cdot \frac{2}{1} = \frac{14}{5}$. \n4. Преобразуем $\frac{14}{5}$ в десятичную дробь: $\frac{14}{5} = 2.8$.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Какое из двух чисел больше: $\frac{7}{4}$ или $1.75$?

Для сравнения числа $\frac{7}{4}$ и $1.75$, преобразуем $\frac{7}{4}$ в десятичную дробь. Разделим 7 на 4, получим $7 \div 4 = 1.75$. Таким образом, $\frac{7}{4} = 1.75$. Поскольку оба значения равны, числа $\frac{7}{4}$ и $1.75$ равны.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Найдите сумму чисел $-3.7$ и $\frac{5}{4}$. Запишите ответ в виде десятичной дроби.

Для решения задачи сначала преобразуем обыкновенную дробь $\frac{5}{4}$ в десятичную дробь. Дробь $\frac{5}{4}$ равна $1.25$. Далее, складываем $-3.7$ и $1.25$: \n\n$$ -3.7 + 1.25 = -3.7 + 1.25 = -2.45 $$
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Фермер собрал с поля 45 тонн картофеля и 30 тонн моркови. В каком отношении масса моркови составляет массу картофеля?

Для того чтобы найти отношение массы моркови к массе картофеля, разделим массу моркови на массу картофеля. Это будет $\frac{30}{45}$. Сократим дробь, разделив числитель и знаменатель на их наибольший общий делитель, который равен 15: $\frac{30 \div 15}{45 \div 15} = \frac{2}{3}$. Следовательно, отношение массы моркови к массе картофеля равно 2:3.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Вычислите значение выражения: $\frac{4}{5} \cdot \left(-\frac{3}{7}\right) \div \frac{2}{3}$.

Для начала произведём умножение: $\frac{4}{5} \cdot \left(-\frac{3}{7}\right) = \frac{4 \cdot (-3)}{5 \cdot 7} = \frac{-12}{35}$. Далее деление заменяется умножением на обратную дробь: $\frac{-12}{35} \div \frac{2}{3} = \frac{-12}{35} \cdot \frac{3}{2}$. После этого умножаем дроби: $\frac{-12 \cdot 3}{35 \cdot 2} = \frac{-36}{70}$. Сократим дробь: $\frac{-36}{70} = \frac{-18}{35}$. Таким образом, значение выражения равно $\frac{-18}{35}$.
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




Решите уравнение: $\frac{3}{x + 2} + 4 = \frac{5}{x + 2} - 2$

Для решения уравнения приведём дроби к общему знаменателю и сравняем их: \n\n1. Преобразуем уравнение: \n$$\frac{3}{x + 2} + 4 = \frac{5}{x + 2} - 2$$\n\n2. Перенесём все члены с дробями в одну сторону: \n$$\frac{3}{x + 2} - \frac{5}{x + 2} = -4 - 2$$\n\n3. Упростим уравнение: \n$$\frac{3 - 5}{x + 2} = -6$$\n\n4. Это выражается как: \n$$\frac{-2}{x + 2} = -6$$\n\n5. Решим уравнение, домножив обе стороны на $(x + 2)$: \n$$-2 = -6(x + 2)$$\n\n6. Раскроем скобки: \n$$-2 = -6x - 12$$\n\n7. Решим относительно $x$: \n$$6x = -12 + 2$$\n$$6x = -10$$\n$$x = -\frac{10}{6} = -\frac{5}{3}$$
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~




{"question": "Лодка движется по реке. Скорость течения реки составляет $2$ км/ч. Лодка проплывает $16$ км по течению и $9$ км против течения, затратив на это $5$ часов. Найдите скорость лодки в неподвижной воде.", "solution": "Пусть $v$ км/ч – скорость лодки в неподвижной воде. Тогда скорость лодки по течению равна $v + 2$ км/ч, а против течения – $v - 2$ км/ч. \\n\\nДля движения по течению:\\n\\\\[ \\\\frac{16}{v + 2} \\\\] – время, затраченное на 16 км.\\n\\nДля движения против течения:\\n\\\\[ \\\\frac{9}{v - 2} \\\\] – время, затраченное на 9 км.\\n\\nОбщее время: \\n\\\\[ \\\\frac{16}{v + 2} + \\\\frac{9}{v - 2} = 5 \\\\]\\n\\nУмножим на $(v + 2)(v - 2)$ и получим уравнение:\\n\\\\[ 16(v - 2) + 9(v + 2) = 5(v^2 - 4) \\\\]\\n\\nРаскрываем скобки и приводим подобные:\\n\\\\[ 16v - 32 + 9v + 18 = 5v^2 - 20 \\\\]\\n\\\\[ 25v - 14 = 5v^2 - 20 \\\\]\\n\\\\[ 0 = 5v^2 - 25v - 6 \\\\]\\n\\nРешим квадратное уравнение: \\n\\\\[ v^2 - 5v - 1.2 = 0 \\\\]\\n\\nИспользуем формулу корней квадратного уравнения:\\n\\\\[ v = \\\\frac{-b \\\\pm \\\\sqrt{b^2 - 4ac}}{2a} \\\\]\\nгде $a = 1$, $b = -5$, $c = -1.2$\\n\\n\\\\[ v = \\\\frac{5 \\\\pm \\\\sqrt{25 + 4.8}}{2} \\\\]\\n\\\\[ v = \\\\frac{5 \\\\pm \\\\sqrt{29.8}}{2} \\\\]\\n\\n\\\\[ v = \\\\frac{5 \\\\pm 5.46}{2} \\\\]\\nВыбираем положительный корень:\\n\\\\[ v = \\\\frac{10.46}{2} = 5.23 \\\\]\\n\\nСкорость лодки в неподвижной воде составляет $5.23$ км/ч.", "topic": "Сложные задачи на движение по реке", "answer": "5.23 км/ч", "type": "Практическая", "tags": ["скорость", "время", "расстояние", "движение", "задача на движение"], "difficulty": "B"}


Пусть $v$ км/ч – скорость лодки в неподвижной воде. Тогда скорость лодки по течению равна $v + 2$ км/ч, а против течения – $v - 2$ км/ч. \\n\\nДля движения по течению:\\n\\\\[ \\\\frac{16}{v + 2} \\\\] – время, затраченное на 16 км.\\n\\nДля движения против течения:\\n\\\\[ \frac{9}{v - 2} \\\\] – время, затраченное на 9 км.\\n\\nОбщее время: \\n\\\\[ \\\\frac{16}{v + 2} + \\\\frac{9}{v - 2} = 5 \\\\]\\n\\nУмножим на $(v + 2)(v - 2)$ и получим уравнение:\\n\\\\[ 16(v - 2) + 9(v + 2) = 5(v^2 - 4) \\\\]\\n\\nРаскрываем скобки и приводим подобные:\\n\\\\[ 16v - 32 + 9v + 18 = 5v^2 - 20 \\\\]\\n\\\\[ 25v - 14 = 5v^2 - 20 \\\\]\\n\\\\[ 0 = 5v^2 - 25v - 6 \\\\]\\n\\nРешим квадратное уравнение: \\n\\\\[ v^2 - 5v - 1.2 = 0 \\\\]\\n\\nИспользуем формулу корней квадратного уравнения:\\n\\\\[ v = \\\\frac{-b \\\\pm \\\\sqrt{b^2 - 4ac}}{2a} \\\\]\\nгде $a = 1$, $b = -5$, $c = -1.2$\\n\\n\\\\[ v = \\\\frac{5 \\\\pm \\\\sqrt{25 + 4.8}}{2} \\\\]\\n\\\\[ v = \\\\frac{5 \\\\pm \\\\sqrt{29.8}}{2} \\\\]\\n\\n\\\\[ v = \\\\frac{5 \\\\pm 5.46}{2} \\\\]\\nВыбираем положительный корень:\\n\\\\[ v = \\\\frac{10.46}{2} = 5.23 \\\\]\\n\\nСкорость лодки в неподвижной воде составляет $5.23$ км/ч.

#### Проблемные задачи:

In [ ]:
# редставьте ответ в виде десятичной дроби
#
# Сколько времени потребуется двум рабочим, чтобы вы
#
# Лодка движется по реке. Скорость течения реки составляе
#


"Пусть \\(x = 5k + 2\\) и \\(x = 7m + 1\\), где \\(k\\) и \\(m\\) — целые числа.\\n\\nПриравняем выражения для \\(x\\):\\n\\n\\(5k + 2 = 7m + 1\\)\\n\\nУпростим уравнение:\\n\\n\\(5k - 7m = -1\\)\\n\\nНайдем частное решение уравнения чисел путем подбора. \\n\\nБерем \\(k = 3\\), тогда:\\n\\n\\(5 \\cdot 3 - 7 \\cdot 2 = 15 - 14 = 1\\)\\n\\nКорректируем на нужный знак:\\n\\n\\(5 \\cdot 1 - 7 \\cdot 1 = -1\\) — корректное решение.\\n\\nТеперь общее решение:\\n\\n\\(5k - 7m = -1\\) — частное решение \\(k = 1, m = 1\\).\\n\\nОбщее решение:\\n\\n\\(k = 1 + 7t\\), \\(m = 1 + 5t\\), где \\(t\\) — любое целое число.\\n\\nПодставим в выражение для \\(x\\):\\n\\n\\(x = 5k + 2 = 5(1 + 7t) + 2 = 5 + 35t + 2 = 7 + 35t\\).\\n\\nПервое положительное число получится при \\(t = 0\\):\\n\\n\\(x = 7\\).